app/vmui/packages/vmui/src/components/CardinalityPanel/consts.ts

```diff
@@ -0,0 +1,46 @@
+export const headCellsWithProgress = [
+  {
+    disablePadding: false,
+    id: "name",
+    label: "Name",
+    numeric: false,
+  },
+  {
+    disablePadding: false,
+    id: "value",
+    label: "Value",
+    numeric: false,
+  },
+  {
+    disablePadding: false,
```

Comment:
Can we add a tooltip explaining `percentage of what` we're looking at?

Reply:
added